package.json

```diff
@@ -20,7 +20,7 @@
     "husky": "^4.2.5",
     "lerna": "^3.20.2",
     "lint-staged": "^10.2.11",
-    "mocha": "^5",
+    "mocha": "^10",
```

Comment:
I bumped mocha to v10. With head of this repo I was getting a runtime error from glob caused by how mocha was calling into glob to find tests. The bump to 10 solved the issue.